### PR TITLE
Fix for FHIR-32886 and FHIR-32887

### PR DESCRIPTION
**Pull request by mlterryMitre**

### CODE REVIEW

File: sushi-config.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ id: hl7.fhir.us.mcode
 canonical: http://hl7.org/fhir/us/mcode
 version: 1.17.1
 name: MinimalCommonOncologyDataElements
-title: "minimal Common Oncology Data Elements (mCODE) Implementation Guide"
+title: "minimal Common Oncology Data Elements (mCODE) Implementation Guide - GDS version"
```
Comment (saulakravitz) [inline sushi-config.yaml:5]: Should this be included in mCODE?

File: input/pagecontent/group-genomics.md [hunk 2]
```diff
@@ -1,10 +1,36 @@
...
+<p>
+This page has been enhanced to include extended guidance on the use of mCODE genomics elements in support of real-world scenarios. 
+</p>
+<ul>
+  <li>Feedback on the FHIR core specification should be submitted to the <a href="http://hl7.org/fhir-issues">FHIR change tracker</a> with "FHIR Core" as the specification.</li>
+  <li>Feedback on the US core profiles should be submitted to the <a href="http://hl7.org/fhir-issues">FHIR change tracker</a> with "US Core" as the specification.</li>
+  <li>Feedback on profiles from one of the other Da Vinci implementation guides should also be submitted to the <a href="http://hl7.org/fhir-issues">FHIR change tracker</a> with the specification set to the appropriate IG.</li>
+</ul>
+</blockquote>
+{% endraw %}
```
Comment (saulakravitz) [inline input/pagecontent/group-genomics.md:12]: Should this be included?

File: ig.ini [hunk 3]
```diff
@@ -1,3 +1,3 @@
 [IG]
 ig = fsh-generated/resources/ImplementationGuide-hl7.fhir.us.mcode.json
-template = #custom-template
+template = fhir.base.template
```
Comment (saulakravitz) [inline ig.ini:3]: Why is the template changing?

**Review by saulakravitz — COMMENTED** [review]
Looks good, modulo my three questions.

File: ig.ini (hunk 3 above)
Comment (mlterryMitre) [reply ig.ini:3]: good catch. this was meant to be a preliminary version for the redesign of biomarkers. I'll change it back.

File: sushi-config.yaml (hunk 1 above)
Comment (mlterryMitre) [reply sushi-config.yaml]: removed GDS version.

File: input/pagecontent/group-genomics.md (hunk 2 above)
Comment (mlterryMitre) [reply input/pagecontent/group-genomics.md]: removed extraneous notes accidentally included with STU2 blockquote.

**Review by saulakravitz — APPROVED** [review]

**Comment by mlterryMitre** [comment]
merging after re-review/approval from @saulakravitz

**Pull request merged**